vllm/model_executor/weight_utils.py

```diff
@@ -132,12 +132,31 @@ def prepare_hf_model_weights(
     if not is_local:
         # Use file lock to prevent multiple processes from
         # downloading the same model weights at the same time.
-        with get_lock(model_name_or_path, cache_dir):
-            hf_folder = snapshot_download(model_name_or_path,
-                                          allow_patterns=allow_patterns,
-                                          cache_dir=cache_dir,
-                                          tqdm_class=Disabledtqdm,
-                                          revision=revision)
+        model_path_temp = os.path.join(
+            os.getenv("HOME"),
+            ".cache/huggingface/hub",
```

Comment:
Please consider the `TRANSFORMERS_CACHE` environment variable here.

See: https://huggingface.co/docs/transformers/installation?highlight=transformers_cache#caching-models

Reply:
You can get more setting for huggingface transformer environment variable here.
https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables